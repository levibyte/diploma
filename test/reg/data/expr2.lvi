var a = 9; 
print a;
if (a == 8) { 
        print "yashik"; 
} else { 
	print a;
	a = 7;
	if ( a == 6 ) {
		print "brnavar";
	}
	print a;  
}

