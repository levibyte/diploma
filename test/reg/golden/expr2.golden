9.000000
9.000000
7.000000
